### ISSUE TITLE
clean up AudioService

### ISSUE THREAD

**Issue by ahmedre** [quran/quran_android#657]
`MediaSessionCompat` and friends give us a lot of stuff that we previously had to do manually. we should clean up `AudioService`. some things on the top of my head:

- remove most of the logic in `onStartCommand` and replace it with corresponding methods in the `MediaSessionCallback` (a few of these are already there to handle bluetooth buttons, like pause/skip/rewind, for example).
- replace current (messy) service/activity communication with `MediaControllerCompat` (see [video](https://youtu.be/XQwe30cZffg?t=2139)).

**Comment by ozbek** [comment]
When done, probably, #628 will also get fixed.

**Comment by murtraja** [comment]
@ahmedre @ozbek I would like to work on this. Is it still relevant?

If yes, can you define the exact task at hand so that it can be broken down into a series of small, logical and sequential commits instead of a big, untrackable and error-prone mess?

**Comment by ahmedre** [comment]
yes, this is still relevant - jazakAllah khairan. sure let me write something this weekend in sha' Allah for the kinds of things i had in mind for this.
